media/history.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" height="28">
```

Comment:
We can use the following codicon instead:
- https://microsoft.github.io/vscode-codicons/dist/codicon.html

Reply:
@vince-fugnitto When I try to use the condicon in the activity bar, it doesn't show up:
![image](https://user-images.githubusercontent.com/23107734/81630515-d2aa9a80-93d3-11ea-8a04-994bedef6743.png)

Reply:
I see that it is not well supported in vscode, and likely not in theia as well.
We could however copy the icon from the codicon repo itself so it is consistent:

- https://github.com/microsoft/vscode-icons/blob/master/icons/light/history.svg
- https://github.com/microsoft/vscode-icons/blob/master/icons/dark/history.svg

Reply:
@vince-fugnitto I think activityBar only accepts one svg icon.
![image](https://user-images.githubusercontent.com/23107734/81745080-24096700-9472-11ea-9166-03ff8610038c.png)

https://github.com/microsoft/vscode-extension-samples/blob/38cf8ca5799b08d9cee6549f8367a9d490b13f8c/tree-view-sample/package.json#L26-L32
I tested vscode's tree-view-sample. The icon was visible for all color themes.

Reply:
> @vince-fugnitto I think `activitybar` only accepts one svg icon

No problem, I just wanted to reference both, I believe we should use this codicon since it aligns with the rest of the application. Do you mind updating the icon to use the codicon and also place a small header (copied from) like we do in the main repository?